pkg/exporter/process.go

```diff
@@ -101,7 +101,7 @@ func InitExportingProcess(collectorAddr net.Addr, obsID uint32, tempRefTimeout u
 func (ep *ExportingProcess) AddSetAndSendMsg(setType entities.ContentType, set entities.Set) (int, error) {
```

Comment:
Is the ExportingProcess in this file a module running on Antrea Agent or Flow Aggregator?
Or both modules share the same code?

Reply:
Yes, they share the same code. Essentially exporting process is considered a library that can be consumed by users in different places.